src/pykeen/evaluation/evaluator.py

```diff
@@ -81,6 +81,7 @@ def __init__(
         batch_size: Optional[int] = None,
         slice_size: Optional[int] = None,
         automatic_memory_optimization: bool = True,
+        mode: Mode = None,
```

Comment:
Should we allow `mode=None`? what does this mean?

Reply:
`mode=None` is a flag for transductive LP. Keeping it `None` helps to trace issues with active graphs in the inductive case